content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
- will be generated for you by the buildpack.
+### Enable/disable HTTPS Redirect
+The HTTPS redirect feature is enabled by default and can be disabled for the
+`httpd` and `nginx` server cases by setting the `BP_PHP_ENABLE_HTTPS_REDIRECT`
+environment variable to `false` at build-time.
+
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_ENABLE_HTTPS_REDIRECT=false
+{{< /code/copyable >}}
 
- You can also provide additional configurations like follows:
+### Configure server admin for HTTPD
+The server admin in the `httpd` case can be overrwritten from the default of
+`admin@localhost` via the `BP_PHP_SERVER_ADMIN` environment variable at
+build-time.
 
 {{< code/copyable >}}
-# buildpack.yml
-php:
-  # directory where web app code is stored
-  # default: htdocs
-  webdirectory: htdocs
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_SERVER_ADMIN=some-admin@localhost
+{{< /code/copyable >}}
+
+### Overwrite the start process
+By default, the buildpacks will determine the right start command to run
+depending on the server that is being used. A user can override the start
+command by using the [Paketo Procfile buildpack][bp/procfile]. Check out the
+[Procfiles]({{< ref "/docs/howto/configuration#procfiles" >}}) section
+of our configuration docs for details on how to use the buildpack.
+
+## Enable a session handler via service bindings
+The PHP Buildpack supports using session handlers for
+[Memcached](https://memcached.org/) and [Redis](https://redis.io/) instances
+via the [PHP Memcached Session Handler Buildpack][bp/memcached] and the [PHP
+Redis Session Handler Buildpack][bp/redis].
+Check out the [PHP session handler documentation][external/php-session-handler]
+for more information on what a PHP session handler is. 
 
-  # directory where library code is stored
-  # default: lib
-  libdirectory: lib
+In order to configure a session handler for either Redis or Memcached the user
+must provide a [service binding]({{< ref "/docs/howto/configuration#bindings"
+>}}).
 
-  # cli script to use
-  # no default
-  script:
+### Enable the session handler for Redis
+To configure a Redis instance session handler, the provided service binding
+should contain the following:
 
-  # default: admin@localhost
-  serveradmin: admin@localhost
+| Binding File | Value |  Required | Description |
+| -------- | -------- | -------- | ----------- |
+| `type`   | `php-redis-session` | yes     | Binding type
+| `host` or `hostname`   | Default: 127.0.0.1 | no     | Redis instance IP address
+| `port`  | Default: 6379 | no     | Redis instance port
+| `password` | Omitted if unset | no     | Redis instance password
+
+The configurations from the service binding are parsed and used to create a
+`php-redis.ini` file with session details, and its path is appended to the
+`PHP_INI_SCAN_DIR` for usage by PHP when the app starts up.
```

Comment:
This implementation info may not be necessary for a user who's just trying to set up redis.
```suggestion
```